presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeJoinQueries.java

```diff
@@ -120,6 +120,30 @@ public void testMergeJoin()
         assertQuery(mergeJoin(), sql, getSession(), sql);
     }
 
+    @Test
+    public void testJoinsWithConstantFilterAndNoCriteria()
+    {
+        // Test double filtered left join with right constant equality.
+        assertQuery("SELECT COUNT(*) FROM (SELECT * FROM lineitem WHERE orderkey % 1024 = 0) "
```

Comment:
The issues you are seeing must be due to bug in serde for "empty" structs. I suggest to modify the query to return a column to unblock and work on this issue separately.